<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BDFacturerUnGroupe</class>
 <widget class="QDialog" name="BDFacturerUnGroupe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Description du groupe</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Groupe :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditGroupe">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="btn_groupe">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>19</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Responsable :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditResponsable">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btn_details">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>19</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Détails des montants</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Frais de location :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditFraisDeLocation">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Frais de restauration :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditFraisDeRestauration">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btn_detailsResto">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/22352-bubka-Print.png</normaloff>:/images/22352-bubka-Print.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Frais de bar :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditFraisDeBar">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btn_detailsBar">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/22352-bubka-Print.png</normaloff>:/images/22352-bubka-Print.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Frais pénalité retard :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditPenaliteRetard">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Total à payer :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditTotalAPayer">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Montant payé:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEditMontantPaye">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Montant restant :</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEditMontantRestant">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Règlement des frais</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rb_regler">
        <property name="text">
         <string>Régler</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="rb_restituer">
        <property name="text">
         <string>Restituer</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Montant à verser :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBoxMontantAVerser">
        <property name="maximum">
         <number>100000000</number>
        </property>
        <property name="singleStep">
         <number>500</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btn_payerTout">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Payer tout</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Mode :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxModeDePaye"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QFormLayout" name="formLayout_2">
     <property name="formAlignment">
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_ok">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_annuler">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
